core/controlplane/config/templates/stack-template.json

```diff
@@ -895,6 +901,50 @@
       }
     },
     {{ end -}}
+    {{ if .LoadBalancer.LoadBalancerV2 }}
```

Comment:
should be `.LoadBalancer.NetworkLoadBalancer`

Reply:
Agreed.